pymongo/collection.py

```diff
@@ -2030,9 +2098,17 @@ def aggregate_raw_batches(self, pipeline, session=None, **kwargs):
                                    explicit_session=session is not None,
                                    **kwargs)
 
-    def watch(self, pipeline=None, full_document=None, resume_after=None,
-              max_await_time_ms=None, batch_size=None, collation=None,
-              start_at_operation_time=None, session=None, start_after=None):
+    def watch(self,
+        pipeline: Optional[_Pipeline] = None,
+        full_document: Optional[str] = None,
+        resume_after: Optional[Mapping[str, Any]] = None,
+        max_await_time_ms: Optional[int] = None,
+        batch_size: Optional[int] = None,
+        collation: Optional[_CollationIn] = None,
+        start_at_operation_time: Optional[Mapping[str, Any]] = None,
```

Comment:
For this and other watch() helpers, operation_time should be a bson Timestamp.

Reply:
Updated

Reply:
Fixed